app/javascript/components/App.js

```diff
@@ -1,15 +1,36 @@
 import React from "react"
 import PropTypes from "prop-types"
-import { BrowserRouter as  Router, Route, Switch, NavLink } from "react-router-dom"
-
-import { Nav, NavItem } from "reactstrap"
+// import { BrowserRouter as  Router, Route, Switch, NavLink } from "react-router-dom"
+//
+// import { Nav, NavItem } from "reactstrap"
 // import "application.scss"
 
-import AboutUs from "./pages/AboutUs"
-import LearnMore from "./pages/LearnMore"
-import Home from "./pages/Home"
+
+import Header from "./components/Header"
 
 class App extends React.Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      apartments: []
+    }
+  }
+
+  //render index with updated apartment array
+  componentDidMount(){
+    this.apartmentIndex()
+  }
+
+  apartmentIndex = () => {
+    fetch("http://localhost:3000/apartments")
```

Comment:
Your fetch can happen to just `"/apartments"`

Reply:
okay